src/core/types/base/botConfig.ts

```diff
@@ -158,3 +177,65 @@ function validateSkipEnvs(envs: NodeJS.ProcessEnv) {
 	assert(envs.SKIP_BID_WALLET, `Please set SKIP_BID_WALLET in env or ".env" file`);
 	assert(envs.SKIP_BID_RATE, `Please set SKIP_BID_RATE in env or ".env" file`);
 }
+/**
+ *
+ */
+async function getIgnoAddresses(chain_id: string, apiurl: string | undefined, ignoreAddresses: any) {
+	if (!apiurl || apiurl.trim() == "") {
+		if (ignoreAddresses) {
+			return ignoreAddresses;
+		} else {
+			return new Set<string>();
+		}
+	}
+	let start = 0;
+	let out: Set<string>;
+	if (ignoreAddresses) {
+		out = ignoreAddresses;
+	} else {
+		out = new Set<string>();
+	}
+	if (apiurl) {
+		const urls: string = apiurl + "addresses?chain_id=" + chain_id;
+		const res1: any = (await axios.get(urls + "&page=1")).data;
+		if (res1.data[0]) {
+			start = Math.ceil(Number(res1.meta.all) / Number(res1.meta.per_page));
+			res1.data.forEach((Element: any) => out.add(Element.address));
+			for (let i = 2; i <= start; i++) {
+				const res = (await axios.get(urls + "?page=" + String(i))).data;
+				res.data.forEach((Element: any) => out.add(Element.address));
+			}
+		}
+	}
+	return out;
+}
+
+/**
+ *
+ */
+async function getRPCfromRegistry(prefix: string, inputurls: any | undefined) {
```

Comment:
no  need to use any here: getRPCfromRegistry(prefix: string, inputurls? : Array<string>){}